CRISPResso2/CRISPRessoPooledCORE.py

```diff
@@ -385,7 +385,7 @@ def main():
             info('Only the bowtie2 reference genome index file was provided. The analysis will be performed using only genomic regions where enough reads align.')
         elif args.bowtie2_index and args.amplicons_file:
             RUNNING_MODE='AMPLICONS_AND_GENOME'
-            info('Amplicon description file and bowtie2 reference genome index files provided. The analysis will be performed using the reads that are aligned only to the amplicons provided and not to other genomic regions.')
+            info('Amplicon description file and bowtie2 reference genome index files provided. Analysis will be performed using reads that are aligned to the amplicons and other genomic regions.')
```

Comment:
I'd undo this change - the analysis will still be only performed on reads that align to locations where provided amplicons align (and not to other genomic regions)